src/js/chartAuthoring.js

```diff
@@ -123,6 +153,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
                                     "onPieChartReady.escalate": "{chartAuthoring}.events.onPieChartReady.fire"
                                 }
                             }
+                        },
+
+                        sonifier: {
+                            type: "floe.chartAuthoring.sonifier",
+                            createOnEvent: "{chartAuthoring}.events.onPieChartReady",
+                            options: {
+                                model: {
+                                    "dataSet": "{dataEntryPanel}.model.dataSet",
+                                    // We relay currentlyPlayingData and isPlaying to the overall component so that sonification play events can be used to change the rest of the interface
+                                    currentlyPlayingData: "{chartAuthoring}.model.currentlyPlayingData",
```

Comment:
Legacy of old implementation hanging around. I looked at how this is done in other cases with relaying the dataset & other "shared" information that needs to go to both the pie & legend, and refactored  so that `pieChart` has an `activeDataId` value relayed to it by the sonifier that it then relays to `activeSliceId` / `activeRowId` of its pie/legend components.